sdk/communication/communication-sms/sample.env

```diff
@@ -1,3 +1,12 @@
-# Used in most samples. Retrieve these values from a Communication Services resource
+# Used in most samples. Retrieve these values from a Communication Services instance
 # in the Azure Portal.
-COMMUNICATION_CONNECTION_STRING="endpoint=<endpoint>;accessKey=<accessKey>"
\ No newline at end of file
+COMMUNICATION_SAMPLES_CONNECTION_STRING="endpoint=https://<resource name>.communication.azure.net/;accessKey=<key>"
+
+# The endpoint to the Communication Services resource
+COMMUNICATION_ENDPOINT="https://<resource name>.communication.azure.net/"
```

Comment:
is this being set in the LiveTest pipeline? should we make the name more in-line with the others? e,g, COMMUNICATION_SAMPLES_ENDPOINT?